src/Data/List/Membership/Setoid/Properties.agda

```diff
@@ -15,14 +15,17 @@ open import Data.List.Relation.Unary.Any as Any using (Any; here; there)
 import Data.List.Relation.Unary.Any.Properties as Any
 import Data.List.Membership.Setoid as Membership
 import Data.List.Relation.Binary.Equality.Setoid as Equality
+import Data.List.Relation.Unary.Unique.Setoid as Unique
 open import Data.Nat using (suc; z≤n; s≤s; _≤_; _<_)
 open import Data.Nat.Properties using (≤-trans; n≤1+n)
 open import Data.Product as Prod using (∃; _×_; _,_ ; ∃₂)
 open import Data.Sum using (_⊎_; inj₁; inj₂)
 open import Function using (_$_; flip; _∘_; id)
 open import Relation.Binary hiding (Decidable)
+  renaming (Irrelevant to Irrelevant₂)
 open import Relation.Binary.PropositionalEquality as P using (_≡_)
 open import Relation.Unary using (Decidable; Pred)
+  renaming (Irrelevant to Irrelevant₁)
 open import Relation.Nullary using (¬_; yes; no)
```

Comment:
Regarding name clashes, we typicall do:

```agda
open import Relation.Binary as B (hiding / using / etc.)
open import Relation.Unary as U (hiding / using / etc.)
```

And then use qualified names to disambiguate: `U.Irrelevant` vs. `B.Irrelevant`. This way we
don't need to have long renaming lists when there are a lot of clashes.